.github/workflows/docs.yml

```diff
@@ -1,20 +1,12 @@
 name: Documentation
 
 on:
+  push:
```

Comment:
Regarding stuff that still needs to be changed on the repo side:
- There is already a prepared gh-pages branch in this repo
- We only need to change the settings in this repo to deploy from that branch instead of using github actions.